src/main/steps/respondent/details-other-proceedings/content.ts

```diff
@@ -1,10 +1,106 @@
+import { isObject } from 'lodash';
+
+import { Checkbox } from '../../../app/case/case';
+import { YesOrNo } from '../../../app/case/definition';
+import { getFilename } from '../../../app/case/formatter/uploaded-files';
 import { TranslationFn } from '../../../app/controller/GetController';
-import { FormContent } from '../../../app/form/Form';
-import {
-  form as applicant2Form,
-  generateContent as applicant2GenerateContent,
-} from '../../applicant2/details-other-proceedings/content';
+import { FormContent, FormFieldsFn } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { generateContent as applicant1GenerateContent } from '../../applicant1/details-other-proceedings/content';
+
+const labels = applicant1Content => ({
+  errors: {
+    applicant2LegalProceedingsDetails: applicant1Content.errors.applicant1LegalProceedingsDetails,
+    applicant2LegalProceedingUploadedFiles: applicant1Content.errors.applicant2LegalProceedingUploadedFiles,
+    applicant2UnableToUploadEvidence: applicant1Content.errors.applicant2UnableToUploadEvidence,
+  },
+});
 
-export const form: FormContent = applicant2Form;
+export const form: FormContent = {
+  fields: userCase => ({
+    applicant2LegalProceedingsDetails: {
+      type: 'textarea',
+      classes: 'govuk-input--width-40',
+      label: l => l.legalProceedingsDetails,
+      labelSize: 'normal',
+      validator: isFieldFilledIn,
+    },
+    applicant2LegalProceedingsConcluded: {
+      type: 'radios',
+      classes: 'govuk-radios',
+      label: l => l.legalProceedingsConcluded,
+      values: [
+        {
+          label: 'Yes',
```

Comment:
To get Welsh translations we could use:
```suggestion
          label: l => l.yes,
```

And similar for label: 'No'

Reply:
Updated